.github/workflows/macos-build.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   build:
-    uses: electron/electron/.github/workflows/macos-pipeline.yml@main
+    uses: ./.github/workflows/macos-pipeline.yml
```

Comment:
Can/should we add this to `linux-build.yml` too? 🙇‍♀️

Reply:
Yup, done 👍